mxcube3/core/components/queue.py

```diff
@@ -1077,19 +1094,13 @@ def set_dc_params(self, model, entry, task_data, sample_model):
                 sample_model
             )
 
-        full_path = os.path.join(
-            HWR.beamline.session.get_base_image_directory(), params.get("subdir", "")
+        full_path, process_path = HWR.beamline.session.get_full_path(
+            params.get("subdir", ""), self.get_folder_tag(params)
```

Comment:
`get_folder_tag` is used to add a tag/name to the run number directory so that each type of collection gets stored in a folder prefixed with the tag. for instance `oscillation_1 ... oscillation_2`, instead of simply `run_1 .. run_2`